Realm/Realm/Realm.cs

```diff
@@ -198,6 +198,7 @@ private static bool IsRealmOpen(string path)
 
         #endregion static
 
+        [SuppressMessage("Usage", "CA2213:Disposable fields should be disposed", Justification = "A realm is responsible for disposing of state as it could be shared")]
```

Comment:
This comment is a bit unclear - it says that "a realm is responsible for disposing", but it attempts to explain why the Realm will not dispose the state. I would suggest rewording it to something like "A State can be shared between multiple Realm instances. It is disposed when the native instance and its BindingContext is destroyed and OnBindingContextDestructedCallback is invoked."

Reply:
Agreed, it should have been "A realm is not responsible"